web/pdf_thumbnail_view.js

```diff
@@ -177,12 +177,11 @@ var PDFThumbnailView = (function PDFThumbnailViewClosure() {
     /**
      * @private
      */
-    _getPageDrawContext: function PDFThumbnailView_getPageDrawContext() {
+    _getPageDrawContext:
+        function PDFThumbnailView_getPageDrawContext(unscaled) {
```

Comment:
Nit: can we perhaps use a slightly more descriptive parameter name here?
What about e.g. `noCtxScale` instead?